`pnpm run docs` cause error

### Describe the bug

The `dist` folder is removed by `dev` command, so `run-p dev docs-dev` always fails. Am I missing any steps? I'm on Windows btw.

```
PS E:\githome\vuejs\vitepress> pnpm run docs

> vitepress@1.0.0-alpha.13 docs E:\githome\vuejs\vitepress
> run-p dev docs-dev


> vitepress@1.0.0-alpha.13 docs-dev E:\githome\vuejs\vitepress
> node ./bin/vitepress dev docs


> vitepress@1.0.0-alpha.13 dev E:\githome\vuejs\vitepress
> rimraf dist && run-s dev-shared dev-start

node:internal/errors:464
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module 'E:\githome\vuejs\vitepress\dist\node\cli.js'
 imported from E:\githome\vuejs\vitepress\bin\vitepress.js
    at new NodeError (node:internal/errors:371:5)
    at finalizeResolution (node:internal/modules/esm/resolve:416:11)
    at moduleResolve (node:internal/modules/esm/resolve:932:10)
    at defaultResolve (node:internal/modules/esm/resolve:1044:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:422:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:222:40)
    at ESMLoader.import (node:internal/modules/esm/loader:276:22)
    at importModuleDynamically (node:internal/modules/esm/translators:111:35)
    at importModuleDynamicallyCallback (node:internal/process/esm_loader:35:14)
    at file:///E:/githome/vuejs/vitepress/bin/vitepress.js:2:1 {
  code: 'ERR_MODULE_NOT_FOUND'
}
 ELIFECYCLE  Command failed with exit code 1.
ERROR: "docs-dev" exited with 1.
```



### Reproduction

Clone the repo, `pnpm i` then `pnpm run docs` . `pnpm run build` does not change anything, since `dist` is removed immediately after running `docs`.

### Expected behavior

It should start the doc's dev server.

### System Info

```shell
System:
    OS: Windows 10 10.0.19044
    CPU: (12) x64 Intel(R) Core(TM) i5-10500 CPU @ 3.10GHz
    Memory: 2.41 GB / 15.72 GB
  Binaries:
    Node: 16.13.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.17 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 8.4.1 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (105.0.1343.27)
    Internet Explorer: 11.0.19041.1566
```



### Additional context

_No response_

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.vuejs.org).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

--- a/package.json
+++ b/package.json
@@ -73,7 +73,7 @@
     "changelog": "conventional-changelog -p angular -i CHANGELOG.md -s",
     "release": "node scripts/release.js",
     "docs": "run-p dev docs-dev",
-    "docs-dev": "wait-on ./dist/node/cli.js && node ./bin/vitepress dev docs",
+    "docs-dev": "wait-on -d 1000 dist/node/cli.js && node ./bin/vitepress dev docs",
     "docs-debug": "node --inspect-brk ./bin/vitepress dev docs",
     "docs-build": "run-s build docs-build-only",
     "docs-build-only": "node ./bin/vitepress build docs",

--- a/package.json
+++ b/package.json
@@ -73,7 +73,7 @@
     "changelog": "conventional-changelog -p angular -i CHANGELOG.md -s",
     "release": "node scripts/release.js",
     "docs": "run-p dev docs-dev",
-    "docs-dev": "node ./bin/vitepress dev docs",
+    "docs-dev": "wait-on ./dist/node/cli.js && node ./bin/vitepress dev docs",
     "docs-debug": "node --inspect-brk ./bin/vitepress dev docs",
     "docs-build": "run-s build docs-build-only",
     "docs-build-only": "node ./bin/vitepress build docs",
@@ -163,7 +163,8 @@
     "supports-color": "^9.2.2",
     "typescript": "^4.8.2",
     "vitest": "^0.23.1",
-    "vue-tsc": "^0.40.7"
+    "vue-tsc": "^0.40.7",
+    "wait-on": "^6.0.1"
   },
   "pnpm": {
     "peerDependencyRules": {

